curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/you-are.md

```diff
@@ -0,0 +1,27 @@
+---
+id: 651dd5296ffb500e3f2ce479
+title: You Are
+challengeType: 22
+dashedName: you-are
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+In English, when making introductions or identifying someone, you use the verb `to be`. In this case, `You are` is used to address the person Maria is talking to and affirmatively identify their occupation.
+
+Maria is introducing herself and confirming Tom's job role. `Are` is used in the present affirmative to make a statement.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Hello, You _ the new graphic designer, right?`
+
+## --blanks--
+
+are
```

Comment:
I can't seem to put the word "Are" into the input.

Reply:
Yea, I noticed - the hotkeys are interfering - specifically the `e`. If you type an `e` quickly after typing another character, it will show up. Or, turn of the keyboard shortcuts. We will have to look into it on another PR.